### ISSUE TITLE
🐛  Logout doesn't work properly?

### ISSUE THREAD

**Issue by danieladugyan** [Dsek-LTH/member-page#932]
Go to a protected page and then logout. You will still have access to the page.

Somewhat related, if your token expires you will still be logged in until you interact with the page.

**Comment by Macludde** [comment]
Isn't this a backend issue? I mean someone having access to a frontend page (that they shouldn't have access to) is not a problem if they are refused access to the protected data by the backend. If, however, they both have access to the protected page AND the protected data, the problem seems to be primarily with the backend, as someone can always overwrite client-side authorization rules if they want to.

**Comment by danieladugyan** [comment]
> Isn't this a backend issue? I mean someone having access to a frontend page (that they shouldn't have access to) is not a problem if they are refused access to the protected data by the backend. If, however, they both have access to the protected page AND the protected data, the problem seems to be primarily with the backend, as someone can always overwrite client-side authorization rules if they want to.

You're right, it's not a problem per se, it's just not very elegant. The backend part correctly restricts users from doing anything they are not authorized to do. You would however expect to get redirected or get some kind of error on the frontend.
